mesh/meshdb.go

```diff
@@ -77,11 +77,18 @@ func (m *MeshDB) GetBlock(id types.BlockID) (*types.Block, error) {
 		return nil, err
 	}
 
-	transactions, err := m.getTransactions(blk.TxIds)
-	if err != nil {
+	txs, missing := m.GetTransactions(blk.TxIds)
+
+	if missing != nil {
```

Comment:
I thought we'll try fetching the txs here ?

Reply:
fetching is done in the sync, this is just to check if theres something missing in our database